v0/destinations/fb/transform.js

```diff
@@ -229,6 +251,10 @@ function responseBuilderSimple(message, payload, destination) {
   const response = defaultRequestConfig();
   response.endpoint = endpoint;
   response.method = defaultPostRequestConfig.requestMethod;
+  response.headers = {
+    // Forward the client IP: https://developers.facebook.com/docs/marketing-api/app-event-api#http
+    "x-forwarded-for": message.context.ip
```

Comment:
```suggestion
    "x-forwarded-for": message?.context.ip || message.request_ip
```
Lets add guard in case of missing context also adding support for `request_ip` which we support

Reply:
Thanks for the review!

A `context` object will always be present, as the message [will be rejected](https://github.com/rudderlabs/rudder-transformer/blob/cb6aac29/v0/destinations/fb/transform.js#L244-L256) if it's missing.

But I applied this change, including the `request_ip` fallback, and added a unit test for it. 👍